### PR TITLE
Add varcheck

### PR DESCRIPTION
**Pull request by tamird**
@BramGruneir @petermattis (for removals from sql/parser)

### CODE REVIEW

File: util/build.go [hunk 1]
```diff
@@ -22,7 +22,6 @@ import "runtime"
 var (
 	// These variables are initialized via the linker -X flag in the
 	// top-level Makefile when compiling release binaries.
-	buildVers string // Go Version
```
Comment (petermattis) [inline util/build.go]: Should we be populating this at build time. Or is it sufficient to get this information from [runtime.Version()](https://golang.org/pkg/runtime/#Version)?

File: sql/parser/select.go [hunk 2]
```diff
@@ -68,10 +68,10 @@ const (
...
 
 // Select.Lock
-const (
-	astForUpdate = " FOR UPDATE"
-	astShareMode = " LOCK IN SHARE MODE"
-)
+// const (
+// 	astForUpdate = " FOR UPDATE"
+// 	astShareMode = " LOCK IN SHARE MODE"
+// )
```
Comment (petermattis) [inline sql/parser/select.go]: You can just delete this. When/if we implement I likely won't use these constants.

**Comment by petermattis** [comment]
LGTM

**Comment by BramGruneir** [comment]
LGTM

**Pull request merged**